packages/nextra-theme-docs/src/footer.tsx

```diff
@@ -94,7 +93,7 @@ export const NavLinks = ({
   )
 }
 
-const Footer: React.FC<PropsWithChildren<{ menu?: boolean }>> = ({ menu }) => {
+const Footer = ({ menu }: { menu?: boolean }): ReactElement => {
```

Comment:
we don't need here `PropsWithChildren` since `<Footer />` never expects receive `children`